x-pack/plugins/search_playground/server/routes.ts

```diff
@@ -305,4 +305,47 @@ export function defineRoutes({
       }
     })
   );
+  router.post(
```

Comment:
suggestion: we could rely on the index management APIs here for getting index mappings instead.

Reply:
For the cases where we need multiple index responses, index management api returns 500. i.e. 'index1,index2' where we expect a response like 
```
{
    index1: ...,
    index2: ...,
}
```

Therefore I will stick with current implementation for this PR but definitely I am open to change it later.

Reply:
yep makes sense!